.github/workflows/basic.yml

```diff
@@ -3,19 +3,27 @@ name: Create pipeline-basic image
 on: workflow_dispatch
 
 jobs:
-  build_basic:
+  build_grpc_c_builder:
     runs-on: ubuntu-latest
-    container:  hashicorp/packer:1.9
+    container:  hashicorp/packer:1.11
```

Comment:
this might cause problems... i hard coded it to 1.9 for a reason because it didn't work in CI or something.

Reply:
I built it already for testing (https://github.com/zepben/ci-utils/actions/runs/10718529904), seems fine.